src/base/bittorrent/torrenthandle.cpp

```diff
@@ -59,6 +59,7 @@
 #include "peerinfo.h"
 #include "trackerentry.h"
 #include "torrenthandle.h"
+#include "gui/torrentcontentmodelitem.h"
```

Comment:
Just this one line (more precisely, what is behind it) can completely BLOCK this PR!!!